src/angular/lightbox/lightbox-container.ts

```diff
@@ -95,4 +104,8 @@ export class SbbLightboxContainer extends _SbbDialogContainerBase {
       focusMonitor
     );
   }
+
+  ngOnInit(): void {
+    this._innerHeight = window.innerHeight;
```

Comment:
This would break server side rendering, where the window (or document) object might not be available.
See e.g. https://github.com/sbb-design-systems/sbb-angular/blob/main/src/angular/autocomplete/autocomplete-trigger.ts#L300-L304